server/filestore.go

```diff
@@ -5749,6 +5749,8 @@ func (fs *fileStore) Snapshot(deadline time.Duration, checkMsgs, includeConsumer
 
 	// We can add to our stream while snapshotting but not delete anything.
 	state := fs.State()
+	var state StreamState
```

Comment:
Don't need above command?

Reply:
Ah bad commit will fix. This file gets messed up by my IDE every time I save it